pkg/version/base.go

```diff
@@ -52,8 +52,8 @@ var (
 	// longer the direct output of "git describe", but a slight
 	// translation to be semver compliant.
 	gitVersion   = "v0.0.0-master+$Format:%h$"
-	gitCommit    = "$Format:%H$"    // sha1 from git, output of $(git rev-parse HEAD)
-	gitTreeState = "not a git tree" // state of git tree, either "clean" or "dirty"
+	gitCommit    = "$Format:%H$" // sha1 from git, output of $(git rev-parse HEAD)
```

Comment:
Can you add a comment here saying that git fills this in automatically? I take it that's what's happening?

Reply:
And maybe explain the mechanism we're using?

Reply:
PTAL